File: GEULGWI-BACKEND/src/main/java/com/lovedbug/geulgwi/core/domain/member/dto/request/SignUpRequest.java
package com.lovedbug.geulgwi.core.domain.member.dto.request;

import com.lovedbug.geulgwi.core.domain.member.constant.MemberGender;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Builder
@NoArgsConstructor
@AllArgsConstructor
@Getter
public class SignUpRequest {
    private String email;
    private String password;
    private String nickname;
    private LocalDateTime birthDate;
    private MemberGender gender;
}

File: GLIM-AOS/app/src/main/java/com/ssafy/glim/core/data/dto/request/SignUpRequest.kt
package com.ssafy.glim.core.data.dto.request

import kotlinx.serialization.Serializable

@Serializable
data class SignUpRequest(
    val email: String,
    val nickname: String,
    val password: String,
    val gender: String,
    val birthDate: List<Int>,
)
